eng/source-build-patches/0001-remove-netcoreapp2.1-targets.patch

```diff
@@ -0,0 +1,54 @@
+From 61101604c89260e680df2d0e52c2dce84629ba00 Mon Sep 17 00:00:00 2001
+From: Tom Deseyn <tom.deseyn@gmail.com>
+Date: Thu, 19 Nov 2020 20:06:23 +0100
+Subject: [PATCH] xliff-tasks: remove netcoreapp2.1 targets
```

Comment:
This patch originally bumped from `netcoreapp2.0` to `net5.0`. I modified it change from `netcoreapp2.1` to `net5.0`. I am not sure if this make sense for `master`, which really means `net6.0`...

Reply:
`master` is still on a `5.0.100` SDK, so I don't think we can use `net6.0` yet. I think this is fine--we'll revisit in stage 4 (prebuilt removal) when presumably the repo will be on a newer SDK.